src/ServicePulse.Host/app/js/services/services.platform-update.js

```diff
@@ -2,30 +2,41 @@
 (function(window, angular, undefined) {
     "use strict";
 
-    function service($http, scConfig) {
-
-        function _getReleases() {
-            return $http
-                .get(scConfig.service_pulse_url, {
-                     responseType: "json"
-                })
-                .then(function(response) {
-                    return response;
-                }, function(response) {
-                    return {
-                        // suppress error
-                        data: []
-                    };
+    function service($http, $q) {
+
+        function getReleases() {
+
+
+            var serviceProductUrls = [
+                { product: 'SP', url: '//platformupdate.particular.net/servicepulse.txt' },
+                { product: 'SC', url: '//platformupdate.particular.net/servicecontrol.txt' }
+            ];
+
+            return $q.all(serviceProductUrls.map(function(item) {
+                    return $http({
+                        method: 'GET',
+                        url: item.url
+                    });
+                }))
+                .then(function(results) {
+                    var resultObj = {};
+                    results.forEach(function(val, i) {
+                        resultObj[serviceProductUrls[i].product] = val.data;
+                    });
+                    return resultObj;
+                }, function () {
```

Comment:
normally I would agree but the purpose of this service is to return something nice for the directive to work with. I don't want to directive making decisions about how to react to network errors etc I only want it to act on the data provided. For now I'm going to leave it this way. When we get to improve SP's overall design we can standardize our approach to this kind of thing.